module-1/classification.js

```diff
@@ -21,8 +21,21 @@ function grade(score) {
      * Also take into consideration the documentation of the function!
      */
     // PLACE YOUR CODE BETWEEN THIS...
-
+    if (score < 0 || score > 100) {
```

Comment:
I would suggest to try to solve the function with the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

Reply:
I rewrote that code, based on your suggestion.

Reply:
I committed and pushed the changes, based our last meeting (classification.js, fibonacci.js).